src/main/java/com/jmb/springfactory/dao/permission/PermissionMySQLServiceImpl.java

```diff
@@ -1,7 +1,15 @@
 package com.jmb.springfactory.dao.permission;
 
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.function.Function;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.function.Function'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)